sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/Generated/AIDocumentIntelligenceModelFactory.cs

```diff
@@ -14,6 +14,30 @@ namespace Azure.AI.DocumentIntelligence
     /// <summary> Model factory for models. </summary>
     public static partial class AIDocumentIntelligenceModelFactory
     {
+        /// <summary> Initializes a new instance of DocumentIntelligenceError. </summary>
+        /// <param name="code"> One of a server-defined set of error codes. </param>
+        /// <param name="message"> A human-readable representation of the error. </param>
+        /// <param name="target"> The target of the error. </param>
+        /// <param name="details"> An array of details about specific errors that led to this reported error. </param>
+        /// <param name="innererror"> An object containing more specific information than the current object about the error. </param>
+        /// <returns> A new <see cref="DocumentIntelligence.DocumentIntelligenceError"/> instance for mocking. </returns>
+        public static DocumentIntelligenceError DocumentIntelligenceError(string code = null, string message = null, string target = null, IEnumerable<DocumentIntelligenceError> details = null, InnerError innererror = null)
```

Comment:
Those methods were just moved up, for some reason. There are no changes in the model factory.